hibernate-osgi/src/test/java/org/hibernate/osgi/test/OsgiIntegrationTest.java

```diff
@@ -144,34 +139,6 @@ private String testingFeatureXmlUrl() {
 	@ProbeBuilder
 	public TestProbeBuilder probeConfiguration(TestProbeBuilder probe) {
 		System.out.println( "Configuring probe..." );
-
-		// Note : I found locally that this part is not needed.  But I am leaving this here as I might
-		// 		someday have a need for tweaking the probe and I want to remember how it is done...
```

Comment:
Let's not remove this. The comment clearly says that the original author wanted us to keep it.